.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -359,29 +364,48 @@
         <GenerateSplashStoryboard
-            Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'"
-            OutputFile="$(_MauiIntermediateStoryboard)"
+            Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'"
+            IntermediateOutputPath="$(_MauiIntermediateSplashScreen)"
             MauiSplashScreen="@(MauiSplashScreen)"
+            InputsFile="$(_MauiSplashInputsFile)"
         />
-        <PropertyGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'">
-            <_MauiStoryboardForPList>$(_MauiIntermediateStoryboard)</_MauiStoryboardForPList>
+        <PropertyGroup Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'">
+            <_MauiIntermediateStoryboard>$(_MauiIntermediateSplashScreen)MauiSplash.storyboard</_MauiIntermediateStoryboard>
+            <_MauiStoryboardForPList>$(_MauiIntermediateSplashScreen)</_MauiStoryboardForPList>
         </PropertyGroup>
-        <ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'">
-            <MauiImage Include="@(MauiSplashScreen)" />
-            <InterfaceDefinition Include="$(_MauiIntermediateStoryboard)" Link="$([System.IO.Path]::GetFileName($(_MauiIntermediateStoryboard)))" />
-            <FileWrites Include="$(_MauiIntermediateStoryboard)" />
+        <ItemGroup Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'">
+            <_MauiSplashAssets Include="$(_MauiIntermediateSplashScreen)**\*" />
+            <_MauiSplashStoryboard Include="$(_MauiIntermediateStoryboard)" />
+            <_MauiSplashImages Include="@(_MauiSplashAssets)" Exclude="@(_MauiSplashStoryboard)" />
+            <!-- Add the launch storyboard file -->
+            <InterfaceDefinition Include="@(_MauiSplashStoryboard)" Link="%(Filename)%(Extension)" />
+            <!-- Add the splash screen images to BundleResource -->
+            <BundleResource Include="@(_MauiSplashImages->'%(FullPath)')">
+                <LogicalName>%(_MauiSplashImages.Filename)%(_MauiSplashImages.Extension)</LogicalName>
+                <TargetPath>%(_MauiSplashImages.Filename)%(_MauiSplashImages.Extension)</TargetPath>
+            </BundleResource>
         </ItemGroup>
+        <!-- If on Windows, using build host, copy the files over to build server host too -->
+        <ItemGroup Condition="'$(BuildSessionId)' != '' and '$(_ResizetizerIsiOSSpecificApp)' == 'True' and '$(IsMacEnabled)'=='true'">
+            <_MauiImagesToCopyToBuildServer Include="@(_MauiSplashAssets)">
+                <TargetPath>%(Identity)</TargetPath>
+            </_MauiImagesToCopyToBuildServer>
+        </ItemGroup>
+        <CopyFilesToBuildServer
+            Condition="'$(BuildSessionId)' != '' and '$(_ResizetizerIsiOSSpecificApp)' == 'True' and '$(IsMacEnabled)'=='true'"
+            SessionId="$(BuildSessionId)"
+            Files="@(_MauiImagesToCopyToBuildServer)" />
```

Comment:
Do @mauroa  and @emaf need to review this part?

Reply:
Yes, that would be preferable.